### ISSUE TITLE
How to add own exceptions to the tokenizer?

### ISSUE THREAD

**Issue by RichStone** [tsproisl/SoMaJo#3]
Hello,

My corpus has some specific vocabulary that I want to handle with the tokenizer. 

For example, I want the string E.ON from my corpus to be handled as a single token.

I have SoMaJo installed via pip and I tried to add this exception in my project's
`MY_PROJECT_PATH/venv/lib/python3.7/site-packages/somajo/single_token_abbreviations_de.txt` file like this:

```bash
...
# Lines starting with “#” are treated as comments and will be ignored.

# EV and charging stations specific tokens
E.ON

Forsch.frage
IT.NRW
...
```

Unfortunately, this does not work like this. 

What would be the best way to handle this?

Do you think it would be a useful functionality to provide an additional option like `somajo-tokenizer add  --single-token-abbr-de-exceptions <exception-list.txt>`

**Comment by tsproisl** [comment]
Thank you for your report!

What you did was entirely correct and that it didn't work was due to some subtle bug. I've just fixed the bug and made a new release (v1.8.3). While I was at it, I also added E.ON to `single_token_abbreviations_de.txt`.

**Comment by RichStone** [comment]
thank you so much for your quick response, everything works perfectly fine with the new version!
